portal-kernel/test/unit/com/liferay/portal/kernel/module/service/SnapshotTest.java

```diff
@@ -268,12 +268,19 @@ public void testNotFoundBundle() {
 			Snapshot<TestService<String>> snapshot = new Snapshot<>(
 				SnapshotTest.class, Snapshot.cast(TestService.class));
 
-			snapshot.get();
+			Assert.assertNull(snapshot.get());
```

Comment:
@W2jx If this commit is to fix the test coverage, please reflect it in the message.